styles.css

```diff
@@ -0,0 +1,91 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  background-color: #07535b;
+}
+
+.header-toolbar {
+  display: flex;
+  flex-direction: row;
+  min-height: 4rem;
+  align-items: center;
+  padding: 15px;
+  color: white;
+  opacity: 1;
+  width: 100%;
+  justify-content: space-between;
+}
+
+.my-logo {
+  color: #fff;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 18px;
+  font-family: Roboto, sans-serif;
+  letter-spacing: 0.03em;
+  text-decoration: none;
+}
+
+.my-logo:hover {
+  text-decoration: none;
+}
+
+.my-logo:active {
+  text-decoration: none;
+}
+
+.my-logo:visited {
+  text-decoration: none;
+}
+
+.content-headline-section {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  background: url("../images/headline_bg.svg");
+  background-repeat: no-repeat;
+  min-height: 40vh;
+}
+
+.content-headline-section > h1 {
+  color: #9ec42b;
+  font-weight: bold;
+  line-height: 72px;
+  font-size: 48px;
+  padding: 1.2rem;
+  font-style: normal;
+  letter-spacing: 0.37px;
+  text-align: center;
+  background-color: transparent;
+  font-family: Roboto, sans-serif;
+}
+
+.content-headline-section > h2 {
+  color: #edecff;
+  font-weight: 500;
+  padding: 1rem;
+  text-align: center;
+  background-color: transparent;
+  font-family: Roboto, sans-serif;
```

Comment:
it should be written like this font-family: 'Roboto', sans-serif; pls add this in body once, no repeated